packages/airnode-admin/src/cli.ts

```diff
@@ -509,7 +509,7 @@ yargs
     }
   )
   .command(
-    'generate-mnemonic',
+    'generate-airnode-mnemonic',
     'Generates a random mnemonic. Uses "ethers.Wallet.createRandom" under the hood',
```

Comment:
We should enrich the description of this command, since now its not clear when to use which one.